app/views/tickets/new.html.erb

```diff
@@ -8,9 +8,11 @@
 
       <div class="datagov_breadcrumb">
         <nav aria-label="Breadcrumb">
-          <li>
-            <%= link_to 'Back', support_path %>
-          </li>
+          <ol>
+            <li>
+              <%= link_to 'Back', support_path %>
```

Comment:
This isn't directly related to your changes here, but perhaps this should be "Support" since we know that will always be the parent page. We're currently saying "Back" is a breadcrumb which isn't strictly true.

Reply:
Sounds sensible. I'd have to check with Blanca. That was the [design](https://projects.invisionapp.com/d/main#/console/10081578/266102083/preview)

Might take a while. So suggest I make a new ticket.

Reply:
👍

Reply:
If we stick with the current design we probably shouldn't give it an aria label of breadcrumb.

Reply:
OK. I'll copy your comments on to the [trello card](https://trello.com/c/BuhLtrXU/241-fix-role-must-be-one-of-the-valid-aria-roles-rolebreadcrumb ) as it was a specific request from the accessibility audit. thanks